test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/ModelBindingHelperTest.cs

```diff
@@ -49,8 +51,7 @@ public class ModelBindingHelperTest
         public async Task TryUpdateModel_ReturnsFalse_IfModelValidationFails()
         {
             // Arrange
-            // Mono issue - https://github.com/aspnet/External/issues/19
-            var expectedMessage = PlatformNormalizer.NormalizeContent("The MyProperty field is required.");
+            var expectedMessage = new RequiredAttribute().FormatErrorMessage("MyProperty");
```

Comment:
Oops, meant that misplaced comment to be here: At least use `ValidationAttributeUtil` to centralize this.